frontend/src/components/Label.js

```diff
@@ -1,6 +1,5 @@
 import styled from 'styled-components/macro'
 
 export default styled.p`
- color: #fac800;
   font-size: 20px;
```

Comment:
It's best to set font-sizes relative to parent font size, this goes all the up to html and body and respects the users font-size settings. Maybe use 1.1em or 1.2em instead?